### PR TITLE
Feat 621 eventing

### PR DESCRIPTION
**Pull request by iuliancorcoja**
# Change or Enhancement

#### Changelog

**New**

- New `TokenAttribute` custom argument to retrieve the token from the request.
- Update `.gitignore` file.

**Changed**

- Replace `getToken` helper method with the custom `TokenAttribute` argument.
- `springfox-swagger2` migrated to version `3.0.0`. New URLs:
  - API docs: http://localhost:7720/api-docs/
  - Swagger UI: http://localhost:7720/swagger-ui/
- Refactor and formatted existing classes to conform to Google Style Guide for Eclipse Java.
  - Style Guide: https://github.com/google/styleguide
  - Profile: https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml

**Removed**

- Unused Maven dependencies (probably a few more unused dependencies left but need to check this).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -115,27 +103,10 @@
...
 			<groupId>io.springfox</groupId>
-			<artifactId>springfox-swagger-ui</artifactId>
-			<version>2.9.2</version>
-		</dependency>
-		<dependency>
-			<groupId>com.google.guava</groupId>
-			<artifactId>guava</artifactId>
-			<version>28.0-jre</version>
+			<artifactId>springfox-boot-starter</artifactId>
+			<version>3.0.0</version>
```
Comment (tlawrie) [inline pom.xml:109]: Nice catch

**Review by tlawrie — CHANGES_REQUESTED** [review]
@iuliancorcoja I checked with @marcusdroy the project is currently using the Google Style for Java with the default 2 spacing for tabs.

Looking at some of the styling changes, this is showing 4 spaces per tab.

**Comment by iuliancorcoja** [comment]
> @iuliancorcoja I checked with @marcusdroy the project is currently using the Google Style for Java with the default 2 spacing for tabs.
> 
> Looking at some of the styling changes, this is showing 4 spaces per tab.

Fixed! Yeah, so Google Style is independent on tab size and type (spaces or tabs). I had this configured in my IDE and changed it to 2 spaces.

**Comment by iuliancorcoja** [comment]
> @iuliancorcoja I checked with @marcusdroy the project is currently using the Google Style for Java with the default 2 spacing for tabs.
> 
> Looking at some of the styling changes, this is showing 4 spaces per tab.

Fixed! Yeah, so Google Style is independent on tab size and type (spaces or tabs). I had this configured in my IDE and changed it to 2 spaces.

**Review by tlawrie — APPROVED** [review]

**Comment by tlawrie** [comment]
Great clean up by @iuliancorcoja with a great addition of the token resolver.

**Pull request merged**